coreos/rkt
# RFC: dbus in stage1

This is rather an RFC than PR, just to see what it takes to have `machinectl login` working. There is rather a lot of ugly stuff I think.

To test it:

1. Build rkt - `RKT_STAGE1_USR_FROM=src RKT_STAGE1_SYSTEMD_VER=v219 ./build`
2. Run rkt - `rkt --insecure-skip-verify --debug run coreos.com/etcd:v2.0.10`
3. Run `machinectl list` to get machine-id (something like `rkt-2160afa2-3b1b-45e8-a1d0-6283169c0bb2`).
4. Run `systemd-run -M <machine-id-from-above> systemctl status dbus.service` - it prints a service name like `run-18541.service`. This runs `systemctl status dbus.service` inside the container.
5. Run `machinectl login <machine-id-from-above>`.
6. Run `systemctl status <service-name-from-above>`.
7. For fun run `journalctl` - nothing interesting there, I think.
8. Press `ctrl+]` three times during a second to terminate the connection to container.
